src/test/java/io/confluent/connect/elasticsearch/integration/ElasticsearchConnectorIT.java

```diff
@@ -171,4 +166,51 @@ public void testUpsert() throws Exception {
       }
     }
   }
+
+  @Test
+  public void testIncompatibleESVersionDataStreamNotSet() throws Exception {
+    container.close();
+    container = ElasticsearchContainer.withESVersion("7.8.1");
+    container.start();
+    setupFromContainer();
+
+    runSimpleTest(props);
```

Comment:
the whole purpose of this test is to make sure the validator fails, why don't you spin up the container in ValidatorTest instead to do this test

Reply:
Resolved in chat. Renamed the test method instead.